libs/architectures/mlpe/architectures/__init__.py

```diff
@@ -1 +1,67 @@
-from .wrapper import architecturize, get_arch_fns
+import inspect
+
+from mlpe.architectures.embeddings import (
+    CoherentDenseEmbedding,
+    Flattener,
+    NChannelDenseEmbedding,
+    ResNet,
+)
+from mlpe.architectures.flows import CouplingFlow, MaskedAutoRegressiveFlow
+
+
+# see the comment in the `embeddings` dict below for an explanation of this
+def _wrap_flow(arch):
+    def func(*args, **kwargs):
+        def f(shape, embedding):
+            return arch(shape, embedding, *args, **kwargs)
+
+        return f
+
+    params = inspect.signature(arch).parameters
+    params = list(params.values())[2:]
+    func.__signature__ = inspect.Signature(params)
+    return func
+
+
+# this is a decorator that takes an embedding
```

Comment:
remove comments

Reply:
In fact let's move the comment as a docstring to the `embeddings`  and `flows` variables below. Add usage for future additions.